ga4gh/frontend.py

```diff
@@ -9,20 +9,30 @@
 
 import os
 import datetime
+import functools
 
 import flask
 import flask.ext.cors as cors
 import humanize
+import yaml
+from flask import session, request
```

Comment:
Can we use `flask.session` and `flask.response` below instead? I think it's useful to know where these identifiers are coming from, especially in a complicated module with lots of imports like this one.